lang/main.json

```diff
@@ -192,7 +192,8 @@
         "transport": "Transport:",
         "transport_plural": "Transports:",
         "bandwidth": "Estimated bandwidth:",
-        "na": "Come back here for connection information once the conference starts"
+        "na": "Come back here for connection information once the conference starts",
+        "direct": " (direct)"
```

Comment:
No biggie, just a suggestion :-) My reasoning, FWIW: P2P is a widely used (and hopefully understood) term in this realm, and without PERC, being in P2P mode means we do have E2E encryption!